<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BilleteraApp</class>
 <widget class="QDialog" name="BilleteraApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0.207, y2:0.954545, stop:0.132159 rgba(156, 178, 178, 255), stop:1 rgba(255, 255, 255, 255))}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt; font-weight:600; color:#ffffff;&quot;&gt;Sign in&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signup_button">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>420</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color:rgb(221, 255, 233);</string>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="user_name_line">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:5px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>User Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email_line">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>260</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:5px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>E-mail</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_line">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:5px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="confirm_password_line">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:5px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Confirm password</string>
    </property>
   </widget>
   <widget class="QLabel" name="create_user_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>270</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>email_line</zorder>
   <zorder>confirm_password_line</zorder>
   <zorder>password_line</zorder>
   <zorder>user_name_line</zorder>
   <zorder>label</zorder>
   <zorder>signup_button</zorder>
   <zorder>create_user_label</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
